### PR TITLE
Add background colour option to tiles operation (#1923)

### PR DESCRIPTION
**Pull request by neave**
I've done the best I can to add background colour as an option for the tile() method. I may have missed something here as I'm not a C++ programmer. Let me know your thoughts. Thanks.

### CODE REVIEW

**Comment by neave** [comment]
Hm it seems the tests fail in regards to skipBlanks, but the PR doesn't touch that code. Any idea what's going on here? Thanks!

**Comment by lovell** [comment]
Thanks for the PR, I think the test failures are due to the need to match the number of channels in the "background" with the number of channels in the image. Try conditionally using `pop_back` to remove the last element when there is no alpha channel.

**Comment by neave** [comment]
OK that seemed to do the trick. It can now set the tiles background colour and passes the tests. I've committed the changes but it's now reporting coverage failure?

File: lib/output.js [hunk 1]
```diff
@@ -603,6 +604,14 @@ function tile (options) {
         throw is.invalidParameterError('angle', 'positive/negative multiple of 90', options.angle);
       }
     }
+    // Background colour
+    if (is.defined(options.background)) {
```
Comment (lovell) [inline lib/output.js]: The `_setBackgroundColourOption` function already contains validation logic so we should be able to use a single, conditionless call to `this._setBackgroundColourOption(options.background)` here.

Comment (neave) [reply lib/output.js]: OK I've tried that but test coverage still seems to fail 🤷‍♂️

Comment (lovell) [reply lib/output.js]: It should be possible to remove the `if (is.defined(...` check too. (I've just spotted a call to `_setBackgroundColourOption` in `lib/resize.js` also contains this unnecessarily-repeated logic, which I'll remove separately.)

Comment (neave) [reply lib/output.js]: OK great, all checks now pass.

**Comment by lovell** [comment]
Brilliant, thank you Paul.

**Comment by caioaugusto1998** [comment]
we report the guy that send emailing to we finally we report the guy

**Comment by ferreiradantas7** [comment]
@lovell hello lovell i need of help with the applications you can say to paul neave

**Pull request merged**